### PR TITLE
PL-26 - Write Encoder

### PR DESCRIPTION
**Pull request by ramilexe**

### CODE REVIEW

File: polywrap/msgpack/write_encoder_test.go [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+	}
+}
+
+func TestWriteI8(t *testing.T) {
+	tests := []struct {
+		name  string
+		input int8
+		want  []byte
+	}{
+		{name: "zero", input: 0, want: []byte{208, 0}},
```
Comment (fetsorn) [inline polywrap/msgpack/write_encoder_test.go:54]: should this be []bytes{0}?

Comment (fetsorn) [inline polywrap/msgpack/write_encoder_test.go]: Should zero be []bytes{0}?

**Review by fetsorn — CHANGES_REQUESTED** [review]
int8 zero should be a single zero byte. 
Also we might want to call the functions WriteInt8 instead of WriteI8, to keep the assemblyscript naming convention. Wasm/rs uses "write_i8" but Rust has "i8" instead of "int8" so it makes more sense there.

P.S. Thanks for awesome work on test cases, a port of this to wasm/as has been merged and wasm/rs is under way.

File: polywrap/msgpack/write_encoder_test.go (hunk 1 above)
Comment (ramilexe) [reply polywrap/msgpack/write_encoder_test.go]: @fetsorn thank you for your report, it is fixed in https://github.com/ConsiderItDone/polywrap-go/pull/2/commits/c3b2aa211a6ac12a5b45b1001b643b37cb64e160

**Pull request merged**